pyproject.toml

```diff
@@ -38,6 +38,9 @@ python = "^3.6.2"
 pyhcl = "^0.4.4"
 requests = "^2.27.1"
 
+# Note on linux krb5 libs are required: `sudo apt install libkrb5-dev` or `yum install krb5-devel`
+requests_gssapi = { version = "^1.2.0", optional = true } # kerberos support, will include `gssapi`
```

Comment:
should this be both here and in the extras? if it's here won't it always install and not just with `hvac[kerberos]`?